content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -13,11 +13,6 @@ To avoid CNI plugin-related errors, verify that you are using or upgrading to a
 container runtime that has been tested to work correctly with your version of
 Kubernetes.
 
-For example, the following container runtimes are being prepared, or have already been prepared, for Kubernetes v1.24:
```

Comment:
suggest having the container runtimes provide a link to a .md support table that will not go stale. Then link from here..

Reply:
or if there is a location in k8s doc space that provides a "tested" table that would suffice.

Reply:
removing completely begs the question.. what has been tested to work... Cheers, Mike

Reply:
hey @mikebrow Are you referring to this type of table https://github.com/containerd/containerd/blob/main/RELEASES.md#kubernetes-support